docs/performance/tuning-search-performance.md

```diff
@@ -0,0 +1,64 @@
+# Tuning Search Performance
+
+ANN is fast, but sometimes it can be improved more by tuning parameters.
+This page shows the essence for improve ANN search by the Vald cluster.
+
+## Tuning Guideline
+
+When the search results do NOT satisfy the expected result, it can be improved by tuning parameters.
+
+First of all, we recommend tuning by following the steps below without doing it blindly.
+
+<div class="mermaid">
+flowchart TD
+    A[Perform Linear Search API]
+    B{Is satisfies?}
+    C[Tuning Parameters to improve precision]
+    D[Tuning Embedding Models]
+    E[Perform Search API]
```

Comment:
Can you consider this style so that the diagram displays on GitHub as well?
https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/

Reply:
@ykadowak could you please check the special note in the PR description? :)

Reply:
That means it won't be displayed on GitHub right? But sometimes you look at the documents on GitHub directly.

Can we use `magefile` to replace ```mermaid with `<div>`?

Reply:
@ykadowak 
the current format can not show correctly on the GitHub.
I will try your suggestion and consider create image instead of mermaid if it won't show as expected.

Reply:
@ykadowak 
I have fixed the HUGO theme for the Vald website.
https://github.com/vdaas/web/pull/158